docker/4.4.0-alpine/healthcheck.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -x -e -u
+
+# Simple healtcheck on BK port
+nc -z localhost ${BK_PORT}
```

Comment:
This check only probes the port. We should have a more thorough probe here. For example with the coommand `bookkeeper shell bookiesanity`, that creates a ledger, writes a few entries, reads them and deletes the ledger.
In that way most critical paths are being exercised.